pkg/project/project.go

```diff
@@ -320,8 +320,6 @@ func fromProjectConfiguration(projectConfig *ProjectConfiguration, fs afero.Fs)
 					relativeServiceEntrypointPath,
 					customRuntime.Dockerfile,
 					customRuntime.Args,
-					// TODO: Get other entrypoint files as ignores
-					[]string{},
```

Comment:
This TODO is actually important, we need to ability to remove additional files outside of the scope of the .dockerignore file so we can do things like removing entrypoints from the scope of other entrypoint files/services.

Reply:
This has now been added to the PR